third_party/silabs/silabs_board.gni

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+declare_args() {
+  # EFR32 board used
+  silabs_board = ""
+}
+
+if (silabs_board == "") {
+  silabs_board = getenv("SILABS_BOARD")
+}
+
+assert(silabs_board != "", "silabs_board must be specified")
+
+# Differentiate between boards
+# - BRD4304A / SLWSTK6000B / MGM12P Module / 2.4GHz@19dBm
```

Comment:
Should the new boards be mentioned here?

Reply:
Yes it should but that applies to all the READMEs. A second PR will be opened thats updates the documentation.